ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -490,3 +490,19 @@ def getUserDict(username):
     if value:
       resDict[key.replace(relPath, '')] = value
   return S_OK(resDict)
+
+
+def getEmailsForGroup(groupName):
+  """ Get email list of users in group
+
+      :param basestring groupName: DIRAC group name
+
+      :return: list
+  """
+  emails = []
+  for username in getUsersInGroup(groupName, defaultValue=[]):
+    email = gConfig.getValue("%s/Users/%s/Email" % (gBaseRegistrySection, username), [])
+    if not isinstance(email, list):
+      email = email.replace(' ', '').split(',')
+    emails.append(email)
```

Comment:
Is this function supposed to return a list of lists? From the docstring I guess this should be:
```suggestion
    emails.extend(email)
```

Reply:
Yes,` list( list(<user1 emails>), list(<user2 emails>).. )`
For example, Notification.sendMail() method can take list of user emails, but send only to one email - first that return success, so in this case:

```
for addresses in Registry.getEmailsForGroup('dirac_admin'):
  result = NotificationClient().sendMail(addresses, 'subject', 'body')

```